web/src/settings_realm_domains.ts

```diff
@@ -9,7 +9,9 @@ import {$t_html} from "./i18n";
 import {page_params} from "./page_params";
 import * as ui_report from "./ui_report";
 
-export function populate_realm_domains_table(realm_domains) {
+export function populate_realm_domains_table(
+    realm_domains: {domain: string; allow_subdomains: boolean}[],
```

Comment:
Let's define this as a `RealmDomain` type.

Reply:
Done. I agree it will be more convenient to just define a type for an often used property.